wordpress-mobile/WordPress-iOS
# Signup Epilogue: Show error if password change fails

Fixes #10268.

This PR makes use of the recent [Account Settings changes](https://github.com/wordpress-mobile/WordPress-iOS/pull/10773) to show an error message if changing the user's password on the signup epilogue screen fails. I've matched the functionality in that PR, so that we either display the error from the API or a generic message if no error is available.

cc @rachelmcr 

**To test:**

* Build and run a clean install (or sign out) – easiest on device due to the magic link email you need to use to sign up.
* Tap **Sign up for WordPress.com**
* Tap **Sign up with Email**
* Enter an email address
* Open your mail account on device and tap the magic email you should've received
* On the screen that now appears in the app, try to enter a short password like 123. You should see a message saying it's too short.
* Try entering a simple password like `hello`. You should see a message saying it's too long.
* Enter a valid password and check it's set correctly.

Update release notes:
